WD-Firmware/rtos/rtosal/src/rtosal_queue.c

```diff
@@ -31,8 +31,11 @@
 #include "rtosal_macro.h"
 #include "rtosal.h"
 #include "psp_api.h"
+#include "psp_attributes.h"
```

Comment:
move psp_attr....h to be in psp_api. 
on the code only psp_api.h should be called